### ISSUE TITLE
NTR: alpha-methylacyl-CoA racemase low in fibroblasts (Lab: blood chemistry: enzymes)

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#7942]
New term request alpha-methylacyl-CoA racemase low in fibroblasts (segal_060322172648): AMACR

**Comment by pnrobinson** [comment]
@MickeySegal
Adding new term: 
- ID: HP:6000134
- label: Reduced alpha-methylacyl-CoA racemase activity in cultured fibroblasts
- parents: [Abnormal enzyme activity in cultured fibroblasts (HP:4000195)](https://hpo.jax.org/app/browse/term/HP:4000195)
- pmid: PMID:10655068
- def: Activity of alpha-methylacyl-CoA racemase (AMACR; EC 5.1.99.4) below the lower limit of normal in cultured fibroblasts.
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/7942
- synonyms:
